pkg/eval/json_evaluator.go

```diff
@@ -274,8 +268,37 @@ func (je *JSONEvaluator) evaluateVariant(
 	return je.state.Flags[flagKey].DefaultVariant, reason, nil
 }
 
+// configToFlags convert string configurations to flags and store them to pointer newFlags
+func (je *JSONEvaluator) configToFlags(config string, newFlags *Flags) error {
+	schemaLoader := gojsonschema.NewStringLoader(schema.FlagdDefinitions)
+	flagStringLoader := gojsonschema.NewStringLoader(config)
+	result, err := gojsonschema.Validate(schemaLoader, flagStringLoader)
+
+	if err != nil {
+		return err
+	} else if !result.Valid() {
+		err := errors.New("invalid JSON file")
```

Comment:
`result.Errors()` provides access to an array of validation errors, this information may be useful for debugging configurations so it may be worth wrapping returned errors, as an example these errors are from the negative tests in the schemas repo:
```
flags.not-a-flag: Must validate one and only one schema (oneOf)
flags.not-a-flag: state is required
```

Reply:
Good suggestion. I did not check the internals as this code came from a refactoring.

I will add detailed logs when this occur 

ex:

`"msg":"JSON schema validation failed:  1:flags.myBoolFlag: Must validate one and only one schema (oneOf)  2:flags.myBoolFlag: state is required  3:flags.myBoolFlag: Must validate all the schemas (allOf) `